test/controllers/versions_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
+
+class VersionControllerTest < ActionDispatch::IntegrationTest
+  test "GET show should be successful" do
+    get "/check.svg"
```

Comment:
Ah yeah of course. I was somehow thinking about it being a binary file. Will do!